assets/scss/styles.scss

```diff
@@ -0,0 +1,406 @@
+$color-blue-light:#3199DC;
+$color-gray-dark: #2A3E51;
+$color-white: white;
+$value-low-space: 30px;
+$value-med-space: 60px;
+$value-hi-space: 100px;
+$font-default: Arial, Helvetica, sans-serif;
+$font-size-xsmall: 10px;
+$font-size-small: 14px;
+$font-size-medium: 18px;
+$font-size-large: 24px;
+$font-site-xlarge: 32px;
+$min-mobile: 320px;
+$max-mobile: 450px;
+$min-tablet: 451px;
+$max-tablet: 1140px;
+
+@mixin mobile {
+  @media (min-width: $min-mobile) and (max-width: $max-mobile) {
+      @content;
+  }
+}
+
+@mixin tablet {
+  @media (min-width: $min-tablet) and (max-width: $max-tablet) {
+      @content;
+  }
+}
+
+@mixin displaynone {
+  display: none;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.cacilds {
+  margin-left: auto;
+  margin-right: auto;
+  font-family: $font-default;
+}
+
+.container {
+  margin-left: auto;
+  margin-right: auto;
+  width: 1140px;
+
+  @include mobile {
+    padding-left: 10px;
+    padding-right: 10px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100%;
+  }
+
+  @include tablet {
+    padding-left: 10px;
+    padding-right: 10px;
+    width: 720px;
+  }
+}
+
+.header {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  background-color: $color-blue-light;
+  margin-bottom: $value-med-space;
+
+  .logo {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &__text {
+      color: $color-white;
+      margin-top: $value-low-space;
+
+      @include mobile {
+        display: none;
+      }
+    }
+
+    &__img {
+      display: block;
+    }
+  }
+
+  .social-medias {
+    @include mobile {
+      display: none;
+    }
+  }
+}
+
+.banner {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: $value-med-space;
+
+  &__container {
+    position: relative;
+  }
+
+  &__title {
+    position: absolute;
+    top: 300px;
+    padding: 0 50px 0 20px;
+    font-size: $font-size-medium;
+    background-color: $color-blue-light;
+    text-transform: uppercase;
+    margin-left: 30px;
+
+    @include mobile {
+      top: 150px;
+      margin-left: 0;
+    }
+  }
+
+  &__subtitle {
+    font-size: $font-size-large;
+    position: absolute;
+    top: 350px;
+    color: $color-white;
+    margin-left: 30px;
+
+    @include mobile {
+      position: static;
+      color: black;
+      margin-left: 0;
```

Comment:
esse é um bom exemplo de um ponto interessante do `mobile first`, nessa declação você precisou zerar o `margin-left` para quando for mobile.

Se a sua aplicação fosse `mobile first`, você não iria precisar declarar esse reset.